manifests/alerting/icingaweb2/director.pp

```diff
@@ -0,0 +1,65 @@
+# This class is used to setup the director module on icingaweb2.
+#
+# It manages all the needed dependencies for setting up the director module
+# and executes the kickstart.
+#
+# @example when including it
+#   class { '::profiles::alerting::icingaweb2::director': }
+#
+# @param db_name      Database name for the director db
+# @param db_username  User for the database
+# @param db_password  The password for the user
+# @param db_host      The host where the database lives
+# @param api_user     User for connecting to the api
+# @param api_password The api user's password
+# @param api_host     The host where the api lives
+class profiles::alerting::icingaweb2::director (
+  String $db_name      = 'director',
+  String $db_username  = 'director',
+  String $db_password  = 'director',
+  String $db_host      = 'localhost',
+  String $api_user     = $::profiles::alerting::icingaweb2::api_user,
+  String $api_password = $::profiles::alerting::icingaweb2::api_password,
+  String $api_host     = 'localhost',
+) inherits profiles::alerting::icingaweb2 {
+
+  postgresql::server::db { $db_name:
+    user     => $db_username,
+    password => $db_password,
+  }
+
+  package { 'git':
+    ensure => present,
+  }
+
```

Comment:
its probably best to place this in manifests/alerting/icingaweb2 in the modules loop as the requirement will hold true for all modules.

also by including it as follows their wont be conflicts with other classes including git e.g. manifests/testing/development 

ensure_resource('package', 'git', {'ensure' => 'present'})